pyproject.toml

```
@@ -18,10 +18,10 @@ classifiers = [
 ]
 
 dependencies = [
-  'numpyro ~= 0.15.0',
+  'numpyro>0.15',
```

This would exclude every (minor) version of numpyro 0.15. It may be better to turn this into

```suggestion
  'numpyro>=0.15',
```

so versions 0.15.x are still supported as in version 0.1.0.